audit/projects/kubernetes-public/services/container/clusters/aaa.json

```diff
@@ -37,8 +37,8 @@
   "binaryAuthorization": {},
   "clusterIpv4Cidr": "10.40.0.0/14",
   "createTime": "2019-09-18T23:39:24+00:00",
-  "currentMasterVersion": "1.21.6-gke.1503",
-  "currentNodeVersion": "1.20.10-gke.1600 *",
+  "currentMasterVersion": "1.21.9-gke.1002",
```

Comment:
manually triggered to catch a fix for GKE. https://github.com/kubernetes/k8s.io/issues/3459#issuecomment-1063754253